builder/steps/gen-dockerfile/contents/test/detect_setup_test.ts

```diff
@@ -40,6 +40,17 @@ const DEFAULT_APP_YAML = 'app.yaml';
 
 const SERVER_JS_CONTENTS = 'echo(\'Hello world\')';
 
+const NODE_VERSION_WARNING = 'WARNING:  Your package.json does not specify ' +
+    'a supported node.js version.  Please pin your application to a major ' +
+    'version of the node.js runtime.  To learn more, visit ' +
+    'https://cloud.google.com/appengine/docs/flexible/nodejs/runtime';
+const NODE_TO_UPDATE_WARNING = 'WARNING: On October 31, 2017, the runtime ' +
```

Comment:
Lets make this less specific. The plan is to update it 'shortly after Node 8 goes LTS'.

Reply:
I've updated the message.